kong/conf_loader/init.lua

```diff
@@ -1339,16 +1346,30 @@ local function load(path, custom_conf, opts)
 
   conf = tablex.merge(conf, defaults) -- intersection (remove extraneous properties)
 
+  local default_nginx_main_user = false
+  local default_nginx_user = false
+
   do
     -- nginx 'user' directive
     local user = utils.strip(conf.nginx_main_user):gsub("%s+", " ")
     if user == "nobody" or user == "nobody nobody" then
       conf.nginx_main_user = nil
+    elseif user == "kong" or user == "kong kong" then
+      default_nginx_main_user = true
```

Comment:
```suggestion

      default_nginx_main_user = true
```

Reply:
I assume you meant this? https://github.com/Kong/kong/pull/6421/files#r540225167

Reply:
https://github.com/Kong/kong/pull/6421/commits/f518f71e88cb05be1508043d22f91e8ee4271185